src/service/poc_mobile.proto

```diff
@@ -156,11 +156,14 @@ message data_transfer_session_req_v1 {
   uint64 upload_bytes = 2;
   uint64 download_bytes = 3;
   data_transfer_radio_access_technology radio_access_technology = 4;
+  string session_id = 5;
+  bytes oui = 6;
```

Comment:
Should it be payer_pub_key? I thought this should be the pub key of the router that pays for traffic (which is attached to OUI).

Reply:
oui doesn't work in the mobile context since getting one requires acquiring slab of lora addresses right @jeffgrunewald ?

If so this would need to become `bytes payer = 6;`.